### PR TITLE
Use MIT license and tempfile, fix rustfmt and Clippy.

### PR DESCRIPTION
**Pull request by afck**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 authors = ["49nord GmbH <marc@49nord.de>"]
```
Comment (mbr) [inline Cargo.toml]: This needs to be `hello@49nord.de`. =)

Comment (mbr) [reply Cargo.toml]: (my bad)

**Review by mbr — APPROVED** [review]
See single change =)

File: Cargo.toml (hunk 1 above)
Comment (afck) [reply Cargo.toml]: Done.
(Very importantly, that also made the commit hash much nicer.)

**Pull request merged**